packages/database/drizzle.config.ts

```diff
@@ -0,0 +1,11 @@
+import type {Config} from 'drizzle-kit';
+import {env} from 'node:process';
+
+export default {
+  schema: './src/db/schema.ts',
+  out: './migrations',
+  driver: 'mysql2',
+  dbCredentials: {
+    connectionString: env.DATABASE_URL || '',
```

Comment:
To be sure: is an empty string allowed? Or should we throw an error if `DATABASE_URL` is missing?

Reply:
It should throw an error. But I could not do this inline. `env!.DATABASE_URL` will still show the typescript error "Type 'string | undefined' is not assignable to type 'string'.". 

I will add this to the top of the file.

```
if (!env.DATABASE_URL) {
  throw new Error('`DATABASE_URL` environment variable is not set');
}
```

Reply:
Perfect! You can also do this: `connectionString: env.DATABASE_URL as string`

Is the `|| ''` still required now?